test/server/cardImplementations/01_SOR/Experience.spec.js

```diff
@@ -0,0 +1,75 @@
+describe('Experience', function() {
+    integration(function() {
+        describe('Experience\'s ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        spaceArena: [{ card: 'cartel-spacer', upgrades: ['experience'] }]
+                    },
+                    player2: {
+                        spaceArena: ['valiant-assault-ship']
+                    }
+                });
+
+                this.cartelSpacer = this.player1.findCardByName('cartel-spacer');
+                this.experience = this.player1.findCardByName('experience');
+                this.assaultShip = this.player2.findCardByName('valiant-assault-ship');
+
+                this.p2Base = this.player2.base;
+            });
+
+            it('should grant the attached unit +1/+1 and be removed from game when unit is defeated', function () {
```

Comment:
Can we add a test that confirms that a cartel spacer with 3 damage and an experience token will die if the experience is removed?

Reply:
Oh good catch. I think that actually will fail right now, I haven't yet implemented the logic that defeats a unit if hp modifiers change in such a way that damage is not >= hp. Will add a TODO here.